src/cloudsc2_ad_loki/CMakeLists.txt

```diff
@@ -110,6 +111,10 @@ if( HAVE_CLOUDSC2_AD_LOKI )
              cloudsc2-common-lib
          DEFINITIONS ${CLOUDSC_DEFINITIONS}
      )
+     # Set specific module directory to avoid aliasing of .mod files
+     set_target_properties( dwarf-cloudsc2-ad-loki-scc
+         PROPERTIES Fortran_MODULE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/loki-scc
+     )
 
      ecbuild_add_test(
          TARGET dwarf-cloudsc2-ad--loki-scc-serial
```

Comment:
Not related to this PR but just noticed: somehow we have a double hyphen in the test name here